.cirrus.yml

```diff
@@ -138,6 +216,19 @@ fedora36_build_task:
     alias: fedora36_build
     depends_on:
       - "build"
+    gce_instance: *standard_gce_x86_64
+    container:
+        cpu: 2
+        memory: 2
+        image: quay.io/libpod/fedora36rust
+    script:
+        - cargo build
+
+fedora36_build_aarch64_task:
```

Comment:
Sorry didn't notice this before.  This task is redundant w/ `build_aarch64_task` (also F36, prev. was F35).